presto-spark-base/src/main/java/com/facebook/presto/spark/execution/nativeprocess/NativeExecutionProcess.java

```diff
@@ -228,6 +231,13 @@ private void populateConfigurationFiles(String configBasePath)
         // to pick unique port per worker to avoid port collision. This config will be passed down to
         // the native execution process eventually for process initialization.
         workerProperty.getSystemConfig().setHttpServerPort(port);
+        // Create a directory on tmpfs for hgdisk spilling use in e2e unit test.
```

Comment:
Is this test-only code? Would it be possible to not include it in production code?